parent/pom.xml

```diff
@@ -2889,6 +2903,7 @@
                 <dep>org.apache.maven.surefire:surefire-junit47</dep>
                 <dep>org.apache.maven.surefire:surefire-junit-platform</dep>
                 <dep>org.codehaus.plexus:plexus-utils</dep>
+                <dep>io.camunda:zeebe-client-java</dep>
```

Comment:
❓ I assume this is also due to the test container artifact depending on Zeebe client?

Reply:
Yes, exactly